setup.cfg

```diff
@@ -23,13 +23,14 @@ classifiers =
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
```

Comment:
Perhaps add 3.11 to CI?

Reply:
Unfortunately the orb CI doesn't work with 3.11 currently

Reply:
Oh? Is there an example I can take a look?

Reply:
https://app.circleci.com/pipelines/github/dwavesystems/dwave-neal/122/workflows/0f58c362-fff1-4ec3-b86d-c5dc4ee831a8

Reply:
That should be an easy fix.

Reply:
IMO we should add CI, but I don't think it needs to block these since we test dwave-samplers with 3.11 already.

Reply:
I agree.